<launch>
    <node pkg="usb_cam" type="usb_cam_node" name="usb_cam_node" output="screen">
        <param name="image_width" type="int" value="640" />
        <param name="image_height" type="int" value="480" />
        <param name="camera_frame_id" type="string" value="cam" />
        <param name="pixel_format" type="string" value="yuyv" />
        <param name="video_device" type="string" value="/dev/video0" />
    </node>
    <node pkg="multi_face_tracker" type="face_detection" name="face_detection">    
        <param name="cam_type" type="string" value="usb_cam" />
        <param name="pyramid_up" type="bool" value="false" />
    </node>
    <node pkg="multi_face_tracker" type="face_ctrl" name="face_ctrl" output="screen">
        <param name="face_landmark_path" type="string" value="$(find multi_face_tracker)/data/shape_predictor_68_face_landmarks.dat" />
        <param name="cam_type" type="string" value="usb_cam" />
        <param name="image_width" type="int" value="640" />
        <param name="image_height" type="int" value="480" />
        <param name="min_accept" type="int" value="2" />
        <param name="min_reject" type="int" value="3" />
    </node>
</launch>
